picard/ui/options/dialog.py

```diff
@@ -93,6 +94,30 @@
 from picard.ui.util import StandardButton
 
 
+class ErrorOptionsPage(OptionsPage):
+
+    def __init__(self, parent=None, errmsg='', from_cls=None):
+        # copy properties from failing page
+        self.NAME = from_cls.NAME
+        self.TITLE = from_cls.TITLE
+        self.PARENT = from_cls.PARENT
+        self.SORT_ORDER = from_cls.SORT_ORDER
+        self.ACTIVE = from_cls.ACTIVE
+        self.HELP_URL = from_cls.HELP_URL
+
+        super().__init__(parent)
+
+        msg = _(
+            "Error while loading option page '%s':\n"
+            "\n"
+            "%s\n"
+            "\n"
+            "Please report this issue."
+        ) % (_(from_cls.TITLE), errmsg)
```

Comment:
What do you think of including a link to the ticket tracker here, similar to the link under "Help" > "Report a bug..." from the menu bar?

Reply:
![image](https://github.com/metabrainz/picard/assets/151042/080b48a9-48df-4812-892f-34e2dedf5d95)